README.md

```diff
@@ -1,52 +1,127 @@
+| Type | Status |
+|---|---|
+| [amd64](https://hub.docker.com/r/amd64/ros/tags) | [![Build Status](https://doi-janky.infosiftr.net/buildStatus/icon?job=multiarch/amd64/ros)](https://doi-janky.infosiftr.net/job/multiarch/job/amd64/job/ros/) |
+| [arm32v7](https://hub.docker.com/r/arm32v7/ros/tags) | [![Build Status](https://doi-janky.infosiftr.net/buildStatus/icon?job=multiarch/arm32v7/ros)](https://doi-janky.infosiftr.net/job/multiarch/job/arm32v7/job/ros/) |
+| [arm64v8](https://hub.docker.com/r/arm64v8/ros/tags) | [![Build Status](https://doi-janky.infosiftr.net/buildStatus/icon?job=multiarch/arm64v8/ros)](https://doi-janky.infosiftr.net/job/multiarch/job/arm64v8/job/ros/) |
 
-* arm32v7 [![Build Status](https://doi-janky.infosiftr.net/buildStatus/icon?job=multiarch/arm32v7/ros)](https://doi-janky.infosiftr.net/job/multiarch/job/arm32v7/job/ros/)
+### [Gazebo ![Docker Pulls](https://img.shields.io/docker/pulls/_/gazebo.svg?label=pulls) ![Docker Stars](https://img.shields.io/docker/stars/_/gazebo.svg?label=stars)](https://hub.docker.com/_/gazebo)
 
+This repo contains images available for Gazebo. For more documentation on using these images, view the Docker Hub repo link above. Images are tagged by releases version, meta package, as well as code name for supported OS base images. For complete listing of tag, view the Repo Info link below.
 
-### [Gazebo ![Docker Pulls](https://img.shields.io/docker/pulls/_/gazebo.svg) ![Docker Stars](https://img.shields.io/docker/stars/_/gazebo.svg)](https://registry.hub.docker.com/_/gazebo/)
-[![Compare Images](https://images.microbadger.com/badges/image/library/gazebo.svg)](https://microbadger.com/#/images/library/gazebo)
+#### [Repo Info](https://github.com/docker-library/repo-info/tree/master/repos/gazebo)
 
-* amd64 [![Build Status](https://doi-janky.infosiftr.net/buildStatus/icon?job=multiarch/amd64/gazebo)](https://doi-janky.infosiftr.net/job/multiarch/job/amd64/job/gazebo/)
+[![Compare Images](https://images.microbadger.com/badges/image/library/gazebo.svg)](https://microbadger.com/#/images/library/gazebo)
 
-* arm64v8 [![Build Status](https://doi-janky.infosiftr.net/buildStatus/icon?job=multiarch/arm64v8/gazebo)](https://doi-janky.infosiftr.net/job/multiarch/job/arm64v8/job/gazebo/)
+#### Architectures
 
-* arm32v7 [![Build Status](https://doi-janky.infosiftr.net/buildStatus/icon?job=multiarch/arm32v7/gazebo)](https://doi-janky.infosiftr.net/job/multiarch/job/arm32v7/job/gazebo/)
+| Type | Status |
+|---|---|
+| [amd64](https://hub.docker.com/r/amd64/gazebo/tags) | [![Build Status](https://doi-janky.infosiftr.net/buildStatus/icon?job=multiarch/amd64/gazebo)](https://doi-janky.infosiftr.net/job/multiarch/job/amd64/job/gazebo/) |
+| [arm32v7](https://hub.docker.com/r/arm32v7/gazebo/tags) | [![Build Status](https://doi-janky.infosiftr.net/buildStatus/icon?job=multiarch/arm32v7/gazebo)](https://doi-janky.infosiftr.net/job/multiarch/job/arm32v7/job/gazebo/) |
```

Comment:
wow this is a sad build history

Reply:
Its been tracked here https://github.com/osrf/docker_images/issues/258 ,
perhaps we could poke it.